### ISSUE TITLE
Ideal batch-size, learning rate for custom dataset with 30k training images

### ISSUE THREAD

**Issue by vizer1993** [fizyr/keras-retinanet#518]
What would be a good batch size to go with a training data of my size? I would like to use all of my 30k images for training. 

With a batch size of 2 (i.e. step size of 15k for 30k images), the estimated time of completion for 1 epoch is shown to be around 7 hours on a Tesla P40. A trade-off between batch-size and time taken per epoch is a priority in my case. 

Also, is it possible to pass an argument to change the default learning rate when invoking train.py ?

**Comment by hgaiser** [comment]
In general a larger batch size produces more stable training, but there is no definite answer as to what is best in your case.

There is no method to configure the learning rate at the moment. It is hardcoded in train.py.
